docs/userguide/data_buildtool.md

```diff
@@ -1,39 +1,67 @@
 ICU Data Build Tool
```

Comment:
Question about file names and organization:

The User Guide so far has used multiple folders (pages with sub-pages) to organize its contents.
https://sites.google.com/site/icuprojectuserguide/system/app/pages/sitemap/hierarchy

Should we recreate this here, or should we use a flat set of files, maybe with topic prefixes like "data_" here?

In Sites, we only have an "icudata" page, but we could decide to make a "data/" folder with pages like overview.md and buildtool.md.

Would a folder structure be a problem for other tools, like automatic index pages or navigation panes?

Reply:
Good thoughts.  My opinion is, let's defer this decision to when the rest of the userguide migration takes place.